pydantic/root_model.py

```diff
@@ -62,10 +62,10 @@ def __init__(self, /, root: RootModelRootType = PydanticUndefined, **data) -> No
             root = data  # type: ignore
         self.__pydantic_validator__.validate_python(root, self_instance=self)
 
-    __init__.__pydantic_base_init__ = True
+    __init__.__pydantic_base_init__ = True  # pyright: ignore[reportFunctionMemberAccess]
 
     @classmethod
-    def model_construct(cls: type[Model], root: RootModelRootType, _fields_set: set[str] | None = None) -> Model:
+    def model_construct(cls: type[Model], root: RootModelRootType, _fields_set: set[str] | None = None) -> Model:  # type: ignore
```

Comment:
What's the problem here / what error is raised?

Reply:
`Method "model_construct" overrides class "BaseModel" in an incompatible manner`, due to the fact that `reportIncompatibleMethodOverride` was recently reported by default to be compliant with the typing spec.